Packs/CortexXpanse/Classifiers/classifier-Xpanse_-_Incoming_Mapper.json

```diff
@@ -6,195 +6,198 @@
+                            }
+                        ]
+                    }
+                },
+                "Xpanse Port": {
+                    "complex": {
+                        "filters": [],
+                        "root": "action_remote_port",
+                        "transformers": []
+                    }
+                },
+                "Xpanse Progress Status": {
+                    "complex": {
+                        "filters": [],
+                        "root": "resolution_status",
+                        "transformers": []
+                    }
+                },
+                "Xpanse Protocol": {
+                    "complex": {
+                        "filters": [],
+                        "root": "port_protocol",
+                        "transformers": []
+                    }
+                },
+                "Xpanse Provider": {
+                    "complex": {
+                        "filters": [],
+                        "root": "cloud_providers",
+                        "transformers": [
+                            {
+                                "operator": "FirstArrayElement"
+                            }
+                        ]
+                    }
+                },
+                "Xpanse Remediation Guidance": {
+                    "complex": {
+                        "filters": [],
+                        "root": "remediation_guidance",
+                        "transformers": []
+                    }
+                },
+                "Xpanse Service ID": {
+                    "complex": {
+                        "filters": [],
+                        "root": "service_ids",
+                        "transformers": [
+                            {
+                                "operator": "FirstArrayElement"
+                            }
+                        ]
+                    }
+                },
+                "Xpanse Tags": {
+                    "complex": {
+                        "filters": [],
+                        "root": "tags",
+                        "transformers": [
+                            {
+                                "args": {
+                                    "applyIfEmpty": {
+                                        "isContext": true
+                                    },
+                                    "defaultValue": {
+                                        "isContext": true,
+                                        "value": {
+                                            "simple": "original_tags"
+                                        }
+                                    }
+                                },
+                                "operator": "SetIfEmpty"
+                            }
+                        ]
+                    }
+                },
                 "Xpanse Category": {
                     "simple": "asm_alert_categories"
+                },
```

Comment:
NIT: This is the only new change in the file

Reply:
"Xpanse Attack Surface Rule Name" is the only change, not sure the entire diff